### ISSUE TITLE
Validation loss does not go down

### ISSUE THREAD

**Issue by vinayakkapoor** [Max-Fu/otter#10]
Hi, I trained otter with the default hyperparameters on one A100 - 80GB and noticed that the validation loss does not consistently go down after a few initial steps (graph attached). I was wondering if it is usual or do I need to change the hyperparameters?

Also, thank you so much for this project, I had a great time going through the pre-print and code! Would love to contribute in any way I can.

![Image](https://github.com/user-attachments/assets/bfa8fc73-0cbc-421e-b32f-57f6025bffd6)
![Image](https://github.com/user-attachments/assets/e74a5c13-7de1-4d5f-9b3d-30b655ed1573)

[config.txt](https://github.com/user-attachments/files/19672794/config.txt)

**Comment by Max-Fu** [comment]
I think this is a usual phenomenon for Otter. We haven't been able to draw much conclusion about overfitting / underfitting from train validation loss comparison.
